### PR TITLE
changelog add date 

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
fixes:

## Description
Added the date of the release to the title of the changelog 
Screenshot 2024-09-15 at 17 16 58

### CODE REVIEW

File: .changelog/4553.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Enhanced the `sdk-changelog` script to include both the version number and release date when generating changelogs for SDK releases.
```
Comment (samuelFain) [inline .changelog/4553.yml:2]: Please use one of the approved templates in [Changelogs](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs).

File: .changelog/4553.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Added support for including release date alongside version number in sdk-changelog script.
```
Comment (dorschw) [inline .changelog/4553.yml:2]:
```suggestion
- description: Future SDK releases will include the release date.
```

File: demisto_sdk/scripts/changelog/changelog.py [hunk 3]
```diff
@@ -279,7 +280,7 @@ def compile_changelog_md(
     # The title
     new_changelog = ["# Changelog"]
     # New version (x.x.x)
-    new_changelog.append(f"## {release_version}")
+    new_changelog.append(f"## {release_version} {datetime.now().strftime('%Y-%m-%d')}")
```
Comment (dorschw) [inline demisto_sdk/scripts/changelog/changelog.py:283]:
```suggestion
    new_changelog.append(f"## {release_version} ({datetime.now().strftime('%Y-%m-%d')})")
```

File: .changelog/4553.yml (hunk 2 above)
Comment (samuelFain) [reply .changelog/4553.yml:2]: @dorschw @shmuel44 
This is still **not** one of the approved templates.
Why not:
```diff
+ - description: Added support for version number and release date when generating changelog entry for an SDK release.
```

Comment (shmuel44) [inline .changelog/4553.yml:2]:
```suggestion
- description: Added support for version number and release date when generating changelog entry for a demisto-sdk release.
```

Comment (shmuel44) [reply .changelog/4553.yml]: Done

File: .changelog/4553.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Added support for version number and release date when generating changelog entry for a demisto-sdk release.
```
Comment (dorschw) [inline .changelog/4553.yml:2]:
```suggestion
- description: Modified the `sdk-changelog` script, to include the release date for each version at the title.
```
@samuelFain wdyt?

**Review by samuelFain — APPROVED** [review]
Nice!

**Pull request merged**